module/core/src/Infrastructure/Service/CurlDownloader.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Core\Infrastructure\Service;
+
+/**
+ */
+class CurlDownloader implements DownloaderInterface
+{
+    private const AGENT = 'Mozilla/5.0 '
+    .'(Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36';
+
+    /**
+     * @param string $url
+     *
+     * @return string|null
+     */
+    public function download(string $url): ?string
+    {
+        $ch = curl_init();
```

Comment:
you should use https://symfony.com/doc/4.4/components/http_client.html

Reply:
Its simple CURL Download at hits moment.
More complicated Downloader will be created in future